monailabel/datastore/utils/convert.py

```
@@ -199,15 +567,56 @@ def itk_image_to_dicom_seg(label, series_dir, template) -> str:
+
+    Returns:
+        Path to output file, or None if conversion fails
+    """
     filename = label if not os.path.isdir(label) else os.path.join(label, os.listdir(label)[0])
 
-    dcm = pydicom.dcmread(filename)
-    reader = pydicom_seg.MultiClassReader()
-    result = reader.read(dcm)
-    image = result.image
+    if not HIGHDICOM_AVAILABLE:
+        raise ImportError("highdicom is not available")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)